doc/source/io.rst

```diff
@@ -573,6 +573,31 @@ take full advantage of the flexibility of the date parsing API:
                     date_parser=conv.parse_date_time)
    df
 
+Pandas will try to call the ``date_parser`` function in three different ways. If 
+an exception is raised, the next one is tried:
+
+1. ``date_parser`` is first called with one or more arrays as arguments,
+   as defined using `parse_dates` (e.g., ``date_parser(['2013', '2013']``, ``['1', '2'])``)
```

Comment:
Here the ``` `` ``` in the middle of the date_parser(..) example should be removed I think